src/app.py

```diff
@@ -142,8 +141,9 @@ def status():
 '''
 
 
-@app.route('/' + app_uuid, methods=["POST"])
-@secured(groups=group_name + "-read")
+# This used to be 'hmuuid' but we are modifying it to be 'uuid'
+@app.route('/uuid', methods=["POST"])
```

Comment:
As Bill suggested, we need to add a second route pattern with the old `/hmuuid` for backward compatibility.